build/ci/release.yml

```diff
@@ -226,30 +225,6 @@ functions:
           - tool_name
         working_dir: src/github.com/mongodb/mongodb-atlas-cli/build/package/docker
         binary: ./docker-build.sh
-  "docker build meta":
-    - command: subprocess.exec
-      type: test
-      params:
-        include_expansions_in_env:
-          - project
-          - revision
-          - created_at
-          - image
-          - tool_name
-        working_dir: src/github.com/mongodb/mongodb-atlas-cli/build/package/docker/meta
-        binary: ./docker-build.sh
-  "powershell build":
-    - command: subprocess.exec
-      type: test
-      params:
-        include_expansions_in_env:
-          - project
-          - revision
-          - created_at
-          - tool_name
-          - unstable
-        working_dir: src/github.com/mongodb/mongodb-atlas-cli/
-        binary: Powershell -File ./build/package/powershell/windows_post_package_test.ps1
```

Comment:
I see this test takes the tool name, are we sure we want to delete for mongocli?

Reply:
The file [mongocli-master/build/package/powershell/windows_post_package_test.ps1](https://github.com/mongodb/mongodb-atlas-cli/blob/mongocli-master/build/package/powershell/windows_post_package_test.ps1) only has references to the atlas cli installer, that's why I think it's safe to remove